src/org/ticket/view/CreateTicket.java

```diff
@@ -35,6 +72,49 @@ public CreateTicket() {
 
     @Override
     public void actionPerformed(ActionEvent e) {
+        Object source = e.getSource();
+
+        if(source.equals(bcreate)) {
+            disable();
+
+            TicketController tc = new TicketController();
+
+            Ticket.TicketPriority tp = Ticket.TicketPriority.LOW;
+            String tps = cbpriority.getSelectedItem().toString();
+
+            if(tps.equals("Média")) {
+                tp = Ticket.TicketPriority.MEDIUM;
+            }
+            else if(tps.equals("Alta")) {
+                tp = Ticket.TicketPriority.HIGH;
+            }
+
+            Ticket t = new Ticket(tftitle.getText(), tfproblem.getText(), false, tp, Session.user);
+
+            Tuple<String, Boolean> resp = tc.save(t);
+
+            if(!resp.second) {
+                JOptionPane.showMessageDialog(null, resp.first, "Erro ao criar ticket",
+                        JOptionPane.ERROR_MESSAGE);
+                tftitle.grabFocus();
+                tftitle.selectAll();
+            } else {
+                JOptionPane.showMessageDialog(null, resp.first, "Sucesso ao criar ticket",
```

Comment:
I believe that this window should be closed and then show this message. If the user wants to open a new he would click on `Novo Chamado` again. To close this window just call:

`frame.dispose()`